lib/http-bridge/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Integration.Scenario.Transactions
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Wallet
+    ( unsafeRunExceptT )
+import Cardano.Wallet.Api.Types
+    ( ApiWallet )
+import Cardano.Wallet.Compatibility.HttpBridge
+    ( HttpBridge )
+import Cardano.Wallet.Network
+    ( NetworkLayer (..) )
+import Cardano.Wallet.Primitive.AddressDerivation
+    ( ChangeChain (..)
+    , FromMnemonic (..)
+    , KeyToAddress (..)
+    , Passphrase
+    , deriveAccountPrivateKey
+    , deriveAddressPrivateKey
+    , generateKeyFromSeed
+    , publicKey
+    )
+import Cardano.Wallet.Primitive.Types
+    ( Address, Coin (..), TxOut (..) )
+import Control.Concurrent
+    ( threadDelay )
+import Data.Text
+    ( Text )
+import Data.Text.Class
+    ( FromText (..) )
+import Test.Hspec
+    ( SpecWith, it )
+import Test.Integration.Faucet
+    ( mkRedeemTx )
+import Test.Integration.Framework.DSL
+    ( Context
+    , Headers (..)
+    , Payload (..)
+    , balanceAvailable
+    , balanceTotal
+    , expectFieldEqual
+    , getFromResponse
+    , json
+    , request
+    , verify
+    , walletId
+    )
+
+import qualified Cardano.Wallet.Network.HttpBridge as HttpBridge
+
+spec :: SpecWith Context
+spec = do
+    -- | NOTE
+    -- This is a temporary test just to see whether we _can_ indeed create
+    -- funded wallets for the integration test scenarios. This will be removed
+    -- and refined into a fixture which will setup a bunch of wallets to work
+    -- with in the integration test.
+    it "Temporary Transaction Test" $ \ctx -> do
+        -- | Create a wallet
+        let payload = Json [json| {
+                "name": "Faucet Wallet #1",
+                "mnemonic_sentence": #{mnemonics},
+                "passphrase": "cardano-wallet"
+                } |]
+        r <- request @ApiWallet ctx ("POST", "v2/wallets") Default payload
+        let wid = getFromResponse walletId r
+
+        -- | Get the first address for that wallet
+        let (Right seed) = fromMnemonic @'[15] mnemonics
+        let (Right pwd) = fromText "cardano-wallet"
+        let addr0 = firstAddress (seed, pwd)
```

Comment:
I suppose It would be actually quite useful to implement the `listAddress` endpoint and use it for this...

Reply:
Hmmm.. Yes, and no. The wallet above aren't in the API _yet_. That's the whole point, we simply create a transaction to the corresponding address, and, if restored, the wallet will already have some funds :)

Reply:
(would be clearer with the next incoming PR, but generally speaking, I do agree about implementing the `listAddress` endpoint. I think I'll tackle that as part of the debt).

Reply:
ok, cool. However it would be nice to have it for the sake of having all the toys in place (for playing with transactions)... Not necessarily in this pr. :)